composer/modules/web/src/ballerina-theme/theme/collections/menu.overrides

```diff
@@ -1,3 +1,24 @@
 /*******************************
          Site Overrides
-*******************************/
\ No newline at end of file
+*******************************/
+
+.ui.menu .item{
```

Comment:
Shall we not override the default menu styles in this particular case since this is a custom menu that we have specifically for design view ?